src/intl/messages.js

```diff
@@ -274,6 +282,7 @@ export const messages: { [messageId: string]: MessageType } = {
   },
   locked: 'Locked',
   logOut: 'Log out',
+  logOutIn30SecondsSecondary: 'You will be logged out in a 30 seconds.',
```

Comment:
"You will be logged out in 30 seconds."

But it would be nice if the number of seconds actually counted down